spec/javascripts/notificationsSpec.js

```diff
@@ -0,0 +1,56 @@
+describe("Notifications", function() {
+  beforeEach(function() {
+    var elements = [];
+    elements.push("<div id='notifications'></div>");
```

Comment:
@julianguyen Yes, we can use the jasmine-jquery library. It will allow us to load fixtures vs. building the dom in jquery. I spiked it out locally, and can push it this weekend.